sycl/test/basic_tests/vectors/vectors.cpp

```diff
@@ -82,5 +109,14 @@ int main() {
   assert((std::is_same<cl::sycl::vec<unsigned long, 8>, cl::sycl::ulong8>::value));
   assert((std::is_same<cl::sycl::vec<unsigned long, 16>, cl::sycl::ulong16>::value));
 
+  // Check convert() from and to various types.
+  check_convert_from<int8_t>();
+  check_convert_from<int16_t>();
+  check_convert_from<int32_t>();
+  check_convert_from<int64_t>();
```

Comment:
Should we have line which `long long` here?

Reply:
On Windows, `std::is_same_v<int64_t,  long long>`, so the test covers the fixed issue.

I guess I can still add `char`, `short`, `int`, `long`, `long long` for more coverage though.